### PR TITLE
Persistence

### PR DESCRIPTION
**Pull request by tigs1995**
- [x] code
- [x] test
- [x] review

### CODE REVIEW

File: src/main/java/com/bae/persistence/domain/Customer.java [hunk 1]
```diff
@@ -1,5 +1,137 @@
...
+	
+	
+	public Customer() {
+	}
+
+	public Customer(String firstName, String lastName, String custoemrEmail, String customerNumber) {
+		super();
+		this.firstName = firstName;
+		this.lastName = lastName;
+		this.customerEmail = custoemrEmail;
```
Comment (nvalydon) [inline src/main/java/com/bae/persistence/domain/Customer.java]: spelt wrong

**Review by nvalydon — APPROVED** [review]
Nice one! Looks good to me :)

**Review by jkiesslinger — APPROVED** [review]
Reviewed by James K

**Review by jkiesslinger — APPROVED** [review]
Amazing well done!

**Review by jkiesslinger — APPROVED** [review]

**Review by jkiesslinger — APPROVED** [review]

**Review by jkiesslinger — APPROVED** [review]
Well done

**Comment by tigs1995** [comment]
Persistence classes updated.

**Pull request merged**